src/Components/People.js

```diff
@@ -0,0 +1,82 @@
+import { Component } from "react";
+import axios from "axios";
+
+import "./People.css"
+
+class People extends Component {
+  state = {
+    input: "",
+    personSearchArray: [],
+    hasSearched: false,
+  };
+
+  handleChange = (e) => {
+    this.setState({ input: e.target.value });
+    this.setState({ hasSearched: true });
+  };
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    this.personSearch();
+  };
+
+  personSearch = async () => {
+    try {
+      const res = await axios.get("https://ghibliapi.herokuapp.com/people");
```

Comment:
Why not grab all the people on the mount, and then on the search just check for the person inside of it? 
Saves an API call and might make breaking it down more straight forward.